changelog/changelog.ddoc

```diff
@@ -8,8 +8,7 @@ CHANGELOG_NAV_FIRST=$(DIVC changelog-nav, next version: $(A $1.html, $1))
 CHANGELOG_NAV_LAST=$(DIVC changelog-nav, previous version: $(A $1.html, $1))
 
 SUBNAV=$(SUBNAV_TEMPLATE
-    $(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions,
-        subnav-duplicate)
+    $(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions)
```

Comment:
Even though it visually looks correct, I don't think this is the right fix.
`SUBNAV_HEAD` is defined in `dlang.org.ddoc` as:
```
SUBNAV_HEAD=
    $(DIVC head,
        $(H2 $1)
        $(TC p, $4,
            $(LINK2 $2, $3))
```
And `TC` is defined in `html.ddoc` as:
```
TC=<$1 class="$2">$(TAIL $+)</$1>
```

I.e. it takes `SUBNAV_HEAD` 4 arguments, where the fourth is the `class` attribute of the `<p>` tag.
Inspecting the generated html reveals that
`$(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions)`  gets expanded to:
```html
<div class="head">
    <h2>Change Log</h2>
    <p class="Change Log, ../changelog/index.html, List of all versions">
        <a href="../changelog/index.html">List of all versions</a>
    </p>
</div>
```

While before
```
    $(SUBNAV_HEAD Change Log, $(ROOT_DIR)changelog/index.html, List of all versions,
        subnav-duplicate)
```

expanded to:

```html
<div class="head">
    <h2>Change Log</h2>
    <p class="        subnav-duplicate">            
        <a href="../changelog/index.html">List of all versions</a>
    </p>
</div>
```

Where `subnav-duplicate` is defined in `css/style.css` as:
```
/* Hide links that just duplicate the subnav contents. They're only displayed
in the narrow layouts without JS. */
.subnav .subnav-duplicate
{
    display: none;
}
```
( Added in commit https://github.com/dlang/dlang.org/commit/736a8bbf3d39cf44da672a62bedc819dc35121f1 )

Hence why the link in the sidebar was not visible.

@aG0aep6G why did you hide link in #1190 (at least it seems intentional to me)?